couchdb/README.md

```diff
@@ -53,30 +53,30 @@ This will create a Secret containing the admin credentials for the cluster.
 Those credentials can be retrieved as follows:
 
 ```bash
-$ kubectl get secret my-release-couchdb -o go-template='{{ .data.adminPassword }}' | base64 --decode
+kubectl get secret my-release-couchdb -o go-template='{{ .data.adminPassword }}' | base64 --decode
 ```
 
 If you prefer to configure the admin credentials directly you can create a
-Secret containing `adminUsername`, `adminPassword` and `cookieAuthSecret` keys:
+Secret containing `adminUsername`, `adminPassword` and `erlangCookie` keys:
```

Comment:
I think this needs both `cookieAuthSecret` (the secret used to encrypt client <-> CouchDB traffic) and `erlangCookie` (the value of the cookie used to authenticate between CouchDB nodes in a cluster). See https://github.com/apache/couchdb-helm/blob/main/couchdb/templates/secrets.yaml.